container/backend/docker/util.go

```diff
@@ -0,0 +1,30 @@
+package docker
+
+import (
+	log "github.com/Sirupsen/logrus"
+	"github.com/docker/engine-api/types"
+	"github.com/opsfactory/kappa/container"
+	"github.com/opsfactory/kappa/container/label"
+	"golang.org/x/net/context"
+)
+
+func NewContainerFromDockerJSON(j types.ContainerJSON) container.Container {
+	c := container.NewContainer()
+	c.Name = j.Name
+	c.Labels = label.NewLabelContainerFromMap(j.Config.Labels)
+	c.NumReplicas = 1
+	c.DesiredReplicas = 1
+	c.Replicas = container.ContainerReplicas{j.ID}
+	c.Backend = container.DockerBackend
+	return c
+}
+
+func (d *Docker) DockerInspect(id string) (container.Container, error) {
```

Comment:
Since this function is already in the `docker` package and beloning to a `Docker` struct do you think that could be a good idea to call it just `Inspect` ?

Real usage scenario of the actual solution:

```
mydocker := docker.SomeFunctionThatReturnsADockerInstance()
mydocker.DockerInspect()
```

Slightly better

```
mydocker := docker.SomeFunctionThatReturnsADockerInstance()
mydocker.Inspect()
```